components/brave_vpn/api/BUILD.gn

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/components/brave_vpn/buildflags/buildflags.gni")
+
+assert(enable_brave_vpn)
+
+source_set("api") {
+  sources = [
+    "brave_vpn_api_helper.cc",
+    "brave_vpn_api_helper.h",
+    "brave_vpn_api_request.cc",
+    "brave_vpn_api_request.h",
+    "vpn_response_parser.cc",
+    "vpn_response_parser.h",
+  ]
+
+  deps = [
+    "//base",
+    "//brave/components/api_request_helper:api_request_helper",
+    "//brave/components/brave_vpn/common",
+    "//brave/components/brave_vpn/mojom",
+    "//brave/components/skus/browser",
+    "//components/prefs",
+    "//net/traffic_annotation",
+    "//services/network/public/cpp",
+    "//url",
+  ]
+}
+
+source_set("unit_tests") {
+  if (enable_brave_vpn) {
```

Comment:
I don't think you need this here, there's already an assert for the entire file so this target can't be included outside of enable_brave_vpn

Reply:
yep, removed